src/strings/csv/en.csv

```diff
@@ -953,6 +953,7 @@ PROJECT_ADDED_ACCESSIONS, {0} accessions,
 PROJECT_ADDED_BATCHES, {0} batches,
 PROJECT_ADDED_DETAILS,Also added the following to {0} project:,
 PROJECT_ADDED_PLANTING_SITES, {0} planting sites,
+PROJECT_EDITED_SUCCESS,Project Updated,
```

Comment:
we use `Changes Saved!` everywhere else.. 
<img width="851" alt="Screen Shot 2024-01-24 at 1 40 13 PM" src="https://github.com/terraware/terraware-web/assets/1865174/53b2f83a-10a4-4c98-ab4f-4ed7ee627561">